
float64 curr_time
dd_bot/DDBotState curr_state
dd_bot/DDBotState[] ref_traj